bootstrap.d/dev-util.yml

```diff
@@ -53,6 +53,25 @@ tools:
     install:
       - args: ['make', 'install']
 
+  - name: host-tzdb-2024a
```

Comment:
I propose we name it host-tzdb. The version 2024a seems redundant here and means we need to update all users if we grab a new version.

Reply:
fair